functional_tests/features/topic_page.feature

```diff
@@ -2,20 +2,19 @@ Feature: CMS users can draft, edit, and publish topic pages
 
     Scenario: A CMS user can feature an article series
         Given a CMS user logs into the admin site
-        And a topic page exists
+        And a topic page exists under a theme page
         And the topic page has a statistical article in a series
         When the user edits the topic page
         And clicks the "Choose Article Series page" button
         And the user selects the article series
         And publishes the page
-        And visits the topic page
+        And the user visits the topic page
         Then the topic page with the example content is displayed
         And the user can see the topic page featured article
 
     Scenario: The feature series on a topic page displays the latest article
-        Given a topic page exists
+        Given a topic page exists under a theme page
         And the user has created a statistical article in a series
         And the user has featured the series
-        When the user creates a new statistical article in the series
```

Comment:
@kacperpONS this condition is valid. The scenario is:
"If there's a featured series that has an article and the user publishes another article in the series, then that last article is features on the topic page"

Reply:
Yup, my bad.
It seems the test step "the user has created a statistical article in a series" doesn't actually create the first article so the test doesn't fully test this functionality. 
Am I misunderstanding something or is that the case? I'm happy to add the extra line.

Reply:
hmm,  it should

```python
@given("the user has created a statistical article in a series")
@given("a statistical article page has been published under the topic page")
def create_article_in_a_series(context: Context):
    context.article_series = ArticleSeriesPageFactory(parent=context.topic_page)
    context.article = StatisticalArticlePageFactory(title="January 2025", parent=context.article_series)
```

Reply:
Have a look at: [45d784b](https://github.com/ONSdigital/dis-wagtail/pull/102/commits/45d784bd05183344e6e2a5433cff89b90796ca4a)
It's also going to need the date to be set explicitly so that it's after the "old" article.